common/constants/default.go

```diff
@@ -90,6 +91,7 @@ const (
 	DefaultCertificateSigningRequestWorkers  = 4
 
 	DefaultUpdatePodStatusBuffer            = 1024
+	DefaultProcessEventBuffer               = 1024
```

Comment:
Please allow me to be curious,

@wbc6080  Do we need to keep `common/constants/default.go` and `staging/src/github.com/kubeedge/api/apis/common/constants/default.go` consistent?

If not, can the unused variables be removed?